src/Tools/SarifValidator/Options.cs

```diff
@@ -20,5 +20,12 @@ internal class Options
             HelpText = "Path of the SARIF file to validate",
             Required = true)]
         public string InstanceFilePath { get; set; }
+
+        [Option(
+            'l',
+            "log-file-path",
+            HelpText = "Path to SARIF log file containing the results of the validation",
+            Default = "sarif.log")]
```

Comment:
I can't do that kind of manipulation in the `Option` attribute declaration because it's not a compile-time constant. How about leaving off the default and I'll compute it by hand in the code?